src/frontend/src/lib/components/send/SendModal.svelte

```diff
@@ -82,6 +86,7 @@
 			bind:targetNetwork
 			bind:amount
 			bind:sendProgressStep
+			formCancelAction={isNullish($pageToken) ? 'back' : 'close'}
```

Comment:
`isNullish($pageToken)` is repeated, please use a local state

Reply:
done!